src/status_im/subs/chats.cljs

```diff
@@ -171,29 +171,26 @@
      chat.events/public-chat?    :public-chat
      :chat)))
 
-(re-frame/reg-sub
- :chats/current-chat-chat-view
+(re-frame/reg-sub :chats/current-chat-chat-view
  :<- [:chats/current-chat]
- (fn [current-chat]
-   (assoc
-    (select-keys current-chat
-                 [:chat-id
-                  :able-to-send-message?
-                  :group-chat
-                  :admins
-                  :invitation-admin
-                  :public?
-                  :chat-type
-                  :color
-                  :contact-request-state
-                  :chat-name
-                  :synced-to
-                  :synced-from
-                  :community-id
-                  :emoji
-                  :description])
-    :empty-chat?
-    (not (:last-message current-chat)))))
+ (fn [chat]
+   {:able-to-send-message? (:able-to-send-message? chat)
```

Comment:
In mobile apps using CLJS I prefer to reserve `select-keys` only for cases where the list of keys is dynamic. Here we know the fixed set of keys, so I extract directly. Destructuring made the code look longer because the destructured keys span over multiple lines, plus the returned map.

Reply:
We know `select-keys` and `merge` are expensive operations, so this change makes a lot of sense :+1: